src/cashier_frontend_new/src/lib/modules/token/services/tokenPrice.svelte.ts

```diff
@@ -0,0 +1,46 @@
+import { HOST_ICP, ICPSWAP_INDEX_CANISTER_ID } from '$lib/constants';
+import type { TokenPrice } from '$lib/modules/token/types';
+import { Actor, HttpAgent } from '@dfinity/agent';
+import * as icpSwapIndexNode from '../../../generated/icpswap/icpswapNodeIndex';
```

Comment:
An absolute path, starting with `$lib/`, is more readable and it should be preferred